EIPS/eip-8078.md

```diff
@@ -0,0 +1,49 @@
+---
+eip: 8078
+title: Hardfork Meta - Heka/Bogota
+description: EIPs included in the Heka/Bogota Ethereum network upgrade.
+author: Tim Beiko (@timbeiko), Alex Stokes (@ralexstokes)
+discussions-to: https://ethereum-magicians.org/t/eip-8078-heka-bogota-network-upgrade-meta-thread/TBD
+status: Draft
+type: Meta
+created: 2025-11-11
+requires: 7723, 7773
+---
+
+## Abstract
+
+This Meta EIP lists the EIPs formally Proposed, Considered, Declined for & Scheduled for Inclusion in the Heka/Bogota network upgrade.
+
+## Specification
+
+Definitions for `Scheduled for Inclusion`, `Considered for Inclusion`, `Declined for Inclusion` and `Proposed for Inclusion` can be found in [EIP-7723](./eip-7723.md).
+
+### EIPs Scheduled for Inclusion
+
+### Considered for Inclusion
+
+### Declined for Inclusion
+
+### Proposed for Inclusion
+
+### Activation
+
+| Network Name | Activation Epoch | Activation Timestamp |
+| ------------ | ---------------- | -------------------- |
+| Sepolia      |                  |                      |
+| Holešky      |                  |                      |
```

Comment:
```suggestion
```
Holešky is deprecated. Pour one out 🥃.
https://blog.ethereum.org/2025/09/01/holesky-shutdown-announcement